bin/export.py

```diff
@@ -242,6 +242,10 @@ def add_file(path, source):
                     f.unlink()
                     f.write_text(text)
 
+    # TODO: Remove this if we know others import the statement folder
+    if (export_dir / "statement").exists():
+        (export_dir / "statement").rename(export_dir / "problem_statement")
```

Comment:
We should probably also pull the contents of `solution/` and `problem_slide/` into `problem_statement/` during the export?

Reply:
tried to implement this :)

Reply:
For some reason, the contents of `solution/` and `problem_slide/` now end up in `statement/` rather than `problem_statement/`? :joy: